TypeScript.YAML-tmTheme

```diff
@@ -42,8 +46,12 @@ settings:
 - scope: entity.name.type.enum.ts
   settings: { vsclassificationtype: enum name }
 
-- scope: entity.name.function, entity.name.type, meta.template.expression.ts, variable, entity.other.inherited-class.ts
+- scope: entity.name.type, meta.template.expression.ts, entity.other.inherited-class.ts
   settings: { vsclassificationtype: identifier }
+- scope: variable, meta.definition.variable.name, support.variable, entity.name.variable, constant.other.placeholder
+  settings: { vsclassificationtype: parameter name }

```

Comment:
how come `const x = 10` x is classified as parameter name ?

Reply:
Fixed typo. This should be `local name`;